### PR TITLE
Adding .ecf files for Stage 1

### PR DESCRIPTION
**Pull request by AarynnCarter**

### CODE REVIEW

File: demos/S1/S1_miri_template.ecf [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+skip_refpix					False
+skip_linearity				False
+skip_rscd					True	#Skipped by default for MIRI TSO
+skip_dark_current			False
+skip_jump					False
+skip_ramp_fitting			False
+skip_gain_scale				False
+
+# Project directory
+topdir						/Users/acarter/Documents/TRANSITS/EUREKA
```
Comment (taylorbell57) [inline demos/S1/S1_miri_template.ecf:25]: Can you upload a copy with generic folder paths like the other ecf templates?

File: eureka/S3_data_reduction/background.py [hunk 2]
```diff
@@ -167,7 +167,7 @@ def fitbg(dataim, meta, mask, x1, x2, deg=1, threshold=5, isrotate=False, isplot
                     residuals = dataslice - model
                     # Simple standard deviation (faster but prone to missing scanned background stars)
                     #stdres = np.std(residuals)
-                    # Median Absolute Deviation (slower but more robust)
+                   # Median Absolute Deviation (slower but more robust)
```
Comment (taylorbell57) [inline eureka/S3_data_reduction/background.py:170]: Why was this space deleted?

File: eureka/lib/astropytable.py [hunk 3]
```diff
@@ -35,6 +35,8 @@ def savetable_S3(filename, bjdtdb, wave_1d, stdspec, stdvar, optspec, opterr):
     opterr = opterr.flatten()
 
     arr = [bjdtdb, wave_1d, stdspec, stdvar, optspec, opterr]
+    arr_shape = [len(a) for a in arr]
```
Comment (taylorbell57) [inline eureka/lib/astropytable.py:38]: Why was this added?

**Review by taylorbell57 — CHANGES_REQUESTED** [review]

**Comment by kevin218** [comment]
Can we get this PR resolved and merged asap?  I've had people asking about missing S1 ecf files.

**Comment by AarynnCarter** [comment]
Will do this first thing tomorrow morning.

**Comment by taylorbell57** [comment]
Also, ideally this PR or another one soon would add the Stage 1 ecfs to the documentation (see the current code in docs/source/ecf.rst for how to format)

File: eureka/lib/astropytable.py (hunk 3 above)
Comment (AarynnCarter) [reply eureka/lib/astropytable.py:38]: This is me debugging things and not tidying up properly, I'll remove it.

**Review by AarynnCarter — COMMENTED** [review]
Changes reviewed and incorporated

File: eureka/S5_lightcurve_fitting/s5_fit.py [hunk 4]
```diff
@@ -95,8 +95,6 @@ def fitJWST(eventlabel, s4_meta=None):
             
             t0 = time.time()
             
-            meta = load_specific_s4_meta_info(old_meta, run_i, spec_hw_val, bg_hw_val)
```
Comment (taylorbell57) [inline eureka/S5_lightcurve_fitting/s5_fit.py:98]: Why was this line deleted? It is a very important line

**Review by taylorbell57 — CHANGES_REQUESTED** [review]
An important line was removed from s5_fit.py that should be added back in

File: eureka/S5_lightcurve_fitting/s5_fit.py (hunk 4 above)
Comment (AarynnCarter) [reply eureka/S5_lightcurve_fitting/s5_fit.py]: There was a github conflict and I think I messed up when fixing it! Should be resolved now

**Review by taylorbell57 — APPROVED** [review]
